OscLib/Constants.h

```diff
@@ -0,0 +1,53 @@
+
+#ifndef OSCLIB_CONSTANTS
+#define OSCLIB_CONSTANTS
+
+#include <cmath>
+
+// \file Constants.h
+// \brief Keep important physical constants and unit conversions in one place.
+//        All values use the natural units system such that hbar = c = 1.
+//        All values are taken from PDG, Physical Constants 1.1, 2024.
+// \author <cullenms@fnal.gov>
```

Comment:
Does Doxygen notice this comment? The safe way:
```suggestion
/// \file Constants.h
/// \brief Keep important physical constants and unit conversions in one place.
///        All values use the natural units system such that hbar = c = 1.
///        All values are taken from PDG, Physical Constants 1.1, 2024.
/// \author <cullenms@fnal.gov>
```